src/overlay/src/Overlay.js

```diff
@@ -181,29 +181,18 @@ class Overlay extends React.Component {
         const autofocusElement = this.containerElement.querySelector(
           '[autofocus]'
         )
+        const wrapperElement = this.containerElement.querySelector('[tabindex]')
+        const buttonElement = this.containerElement.querySelector('button')
 
-        // eslint-disable-next-line eqeqeq, no-eq-null, no-negated-condition
+        // eslint-disable-next-line eqeqeq, no-eq-null
         if (autofocusElement != null) {
```

Comment:
You should use `if (autofocusElement) {` instead and remove all the `eslint-disable-next-line`'s.